.github/workflows/audit_service_tags.yml

```diff
@@ -45,20 +45,19 @@ jobs:
       - name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v3
 
-      - name: Build Docker Image
-        uses: docker/build-push-action@v6
+      - name: Setup Caching for Docker Compose
+        uses: yu-ichiro/spin-up-docker-compose-action@main
```

Comment:
Did Eric recommend changing this to not be `main`?

Reply:
Yes! Although we did decide to go another way with this. The commit hashes should be there for the other gh-action that we're using instead.